lib/views/schedule/schedule.dart

```diff
@@ -12,59 +14,48 @@ class Schedule extends StatelessWidget {
     return Scaffold(
       appBar: AppBar(
         title: const Text('Livestream Schedule'),
-        backgroundColor: ThcColors.darkBlue,
```

Comment:
`ThcColors.darkBlue` is the [default AppBar color](https://github.com/theheartcenter-one/thc/blob/b7c857a3c32806ebb34440ebdebdd3a0fff1990f/lib/models/theme.dart#L115), so we're good to remove this.